monai/transforms/utils.py

```diff
@@ -891,16 +892,20 @@ def generate_spatial_bounding_box(
         [1st_spatial_dim_start, 2nd_spatial_dim_start, ..., Nth_spatial_dim_start],
         [1st_spatial_dim_end, 2nd_spatial_dim_end, ..., Nth_spatial_dim_end]
 
-    The bounding boxes edges are aligned with the input image edges.
-    This function returns [-1, -1, ...], [-1, -1, ...] if there's no positive intensity.
+    If `align_image_edge`, the bounding boxes edges are aligned with the input image edges.
+    This function returns [0, 0, ...], [0, 0, ...] if there's no positive intensity.
```

Comment:
https://github.com/Project-MONAI/MONAI/blob/fd63af4c9082854bc4ad0b01990b3747aa77970e/monai/transforms/croppad/array.py#L1186 this also use the util

Reply:
Hi @wyli ,

I added it to this transform and the DeepGrow transform in the latest commit.

Thanks.